src/fluent-plugin-mdsd/lib/fluent/plugin/out_mdsd.rb

```diff
@@ -19,6 +19,10 @@ def initialize()
         config_param :djsonsocket, :string
         desc 'if no ack is received from mdsd after N milli-seconds, drop msg.'
         config_param :acktimeoutms, :integer
+        desc 'Fluentd tag prefix that will be treated as mdsd syslog messages'
+        config_param :mdsd_syslog_tag_prefix, :string, default: nil
+        desc 'If true, change (unify) any tags starting with the prefix to the prefix only'
+        config_param :unify_mdsd_syslog_tags, :bool, default: false
```

Comment:
This is intentional, because for the extended syslog uploading scenario (where each facility.level can be assigned a unique table), we wouldn't want to unify the tags, and yet we might want to reformat the record for our mdsd syslog purpose (this code will come later here--I thought here is the right place for that). So that's why I intentionally introduced this flag.